openai/openai.go

```diff
@@ -266,7 +268,7 @@ func toChatCompletion(id string, r api.ChatResponse) ChatCompletion {
 	}
 }
 
-func toChunk(id string, r api.ChatResponse) ChatCompletionChunk {
+func toChunk(id string, r api.ChatResponse, toolCallsSent bool) ChatCompletionChunk {
```

Comment:
Instead of modifying this signature we should be able to do this with an if outside